DXMainClient/Domain/Multiplayer/Map.cs

```diff
@@ -711,11 +712,15 @@ public Texture2D LoadPreviewTexture()
 
         public IniFile GetMapIni()
         {
-            var mapIni = new IniFile(CompleteFilePath);
+            Encoding encoding = FileHelper.GetEncoding(CompleteFilePath);
+
+            var mapIni = new IniFile(CompleteFilePath, encoding);
 
             if (!string.IsNullOrEmpty(ExtraININame))
             {
-                var extraIni = new IniFile(SafePath.CombineFilePath(ProgramConstants.GamePath, "INI", "Map Code", ExtraININame));
+                string extraIniPath = SafePath.CombineFilePath(ProgramConstants.GamePath, "INI", "Map Code", ExtraININame);
+                encoding = FileHelper.GetEncoding(extraIniPath);
```

Comment:
Is it TS the only platform that requires a non Unicode encoding on this file? I don't remember the spawner for YR or Phobos has this requirements. Therefore, I suggest wrapping it with an `if-TS`.

Detecting file encoding from a text file is basically "guessing", so we should avoid such a behavior if we are certain the program that processes the text file in UTF8, with or without BOM.

Reply:
Done by 0435e605f66a171bacc63843ea251f02969af45e.

Reply:
The file helper should provide the general, game-irrelevant methods.

If-TS should be applied directly here, instead of in the ClientCore

Reply:
some YR mods still rely on non-Unicode for translating their stuff, and it is possible in principle (albeit is more problematic than unicode, but Unicode input is only propely supported in Phobos, so there's that), so I don't think that is valid necessarily